scripts/check_whitespace.sh

```diff
@@ -29,14 +29,16 @@ git grep "${options[@]}" -- \
     ':(exclude)Firebase/CoreDiagnostics/FIRCDLibrary/Protogen/nanopb' \
     ':(exclude)Firebase/CoreDiagnostics/ProtoSupport' \
     ':(exclude)CoreOnly/NOTICES' \
+    ':(exclude)Firebase/Firebase/NOTICES' \
```

Comment:
Still catching up, but we really should avoid a third NOTICES file.

Reply:
removed